### ISSUE TITLE
Manage tokens unexpected outcome with a motion

### ISSUE THREAD

**Issue by arrenv** [JoinColony/colonyCDapp#2713]
## Steps to reproduce

1. Create a Manage tokens action using permissions decision method.
2. Add and remove some tokens, I ADDED "USDT for Local Development" and "DAI for Local Development", and I REMOVED "Gotham Guilder".
![image](https://github.com/user-attachments/assets/0e91be5a-470a-43f6-8a65-3eeff16eee4f)
3. Create a Manage tokens action using Reputation decision method.
4. Add and remove some tokens, I ADDED "USDC for Local Development" and "Gotham Guilder", and I REMOVED "DAI for Local Development". Don't finalize the motion.
![image](https://github.com/user-attachments/assets/d1e0c1c2-c463-48e5-825b-4bc3968f9a26)
5. Create a Manage tokens action using permissions decision method.
6. Remove some tokens, I REMOVED "DAI for Local Development" and "USDT for Local Development".
![image](https://github.com/user-attachments/assets/bdfa49f4-0556-4bcb-b1aa-a9927e33ed1c)
7. Return to the motion and finalize it.
8. The state of the managed tokens does not match the expected state.

So, while "USDT for Local Development" was removed in the 3rd action using permissions before the motion was finalized, and the motion did not explicitly add "USDT for Local Development", it was still added by the motion.
 

## Expected behaviour

* Motions should only make the proposed changes to the state of managed tokens.
* Expected token state:
    - Ether
    - Space Credits
    - USDC for Local Development
    - Gotham Guilder

## Actual behaviour

* The state of the managed tokens did not match expectations.
* Tokens that should have been removed, were added back in after motion finalization.
* Actual token state:
    - Ether
    - Space Credits
    - USDC for Local Development
    - USDT for Local Development
    - Gotham Guilder

    ![image](https://github.com/user-attachments/assets/144e8bbe-867e-43fd-97cb-3c7f2c1176fc)

**Comment by iamsamgibbs** [comment]
Thanks for raising this @arrenv - I have a suspicion a similar issue will exist with verified members as it works in a similar way.

**Comment by rdig** [comment]
Correct. This was always known to be a problem. And it's not just here. 

Pretty much everywhere we can update metadata via contract calls, and more specifically motions. So even colony details / objective, etc is affected

And it will only get worse with multicall
